src/main/java/com/sharewalk/model/User.java

```diff
@@ -56,4 +56,16 @@ public void setPassword(String password) {
     public String toString() {
         return "User [id=" + id + ", name=" + email +"]";
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Where do you need those equals methods? Don't add such things 'for the future'. It only makes code more complex. In programming there is even a rule called YAGNI - https://en.wikipedia.org/wiki/You_aren't_gonna_need_it